packages/@aws-cdk/aws-stepfunctions/package.json

```diff
@@ -88,21 +88,8 @@
   },
   "awslint": {
     "exclude": [
-      "import-props-interface:@aws-cdk/aws-stepfunctions.ActivityImportProps",
-      "export:@aws-cdk/aws-stepfunctions.IActivity",
-      "duration-prop-type:@aws-cdk/aws-stepfunctions.WaitProps.duration",
       "duration-prop-type:@aws-cdk/aws-stepfunctions.Errors.TIMEOUT",
-      "no-unused-type:@aws-cdk/aws-stepfunctions.ServiceIntegrationPattern",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.TaskInput.value",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.InputType.OBJECT",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.Data.isJsonPathString",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.Result.value",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.StateGraph.startState",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.TaskInput.type",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.InputType.TEXT",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.WaitTime",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.ActivityProps",
-      "docs-public-apis:@aws-cdk/aws-stepfunctions.IActivity"
+      "no-unused-type:@aws-cdk/aws-stepfunctions.ServiceIntegrationPattern"
```

Comment:
(Not related to this PR)

I'm wondering - if this class isn't used anywhere in this package, should it instead be moved to the stepfunction-tasks module?

Reply:
that was my thinking as well. I'm intending to include that in another PR (there's a few other things that need to be cleaned up between these 2 packages)

Reply:
> there's a few other things

can we capture this 'list' somewhere? I would suggest opening an issue, listing it down and linking it off StepFunctions tracking issue.